.cicd/platforms/unpinned/ubuntu-18.04-unpinned.dockerfile

```diff
@@ -24,10 +25,17 @@ RUN curl -LO https://dl.bintray.com/boostorg/release/1.71.0/source/boost_1_71_0.
     ./b2 --with-iostreams --with-date_time --with-filesystem --with-system --with-program_options --with-chrono --with-test -j$(nproc) install && \
     cd / && \
     rm -rf boost_1_71_0.tar.bz2 /boost_1_71_0
-# install eosio cppkin
-RUN curl -LO https://github.com/EOSIO/cppKin/releases/download/v1.2.0/cppkin_1.2.0-1-ubuntu-18.04_amd64.deb && \
-    apt install -y ./cppkin_1.2.0-1-ubuntu-18.04_amd64.deb && \
-    rm ./cppkin_1.2.0-1-ubuntu-18.04_amd64.deb
+# build eosio cppkin
+RUN git clone --single-branch --branch remove-core https://github.com/EOSIO/cppKin.git && \
```

Comment:
This probably should get updated to master as well.

Reply:
Thanks, missed changing that somehow.